resources_pages/install_ds_stack_mac.md

```diff
@@ -318,7 +324,7 @@ The spellchecker helps us correcting typos in our writing.
 Install them via the following commands:
 
 ```bash
-conda install pandas memory_profiler jupyterlab jupyterlab-git jupyterlab-spellchecker jupytext  
```

Comment:
@quan3010 Just a ping to double check that you are not using the `memory_profiler` in 511. I vaguely remember it being used in that class some years ago, but not sure if it was really important or is used at all now.

Reply:
LGTM. I am not using memory_profiler